ee-feature-pack/common/src/main/resources/modules/system/layers/base/org/infinispan/protostream/main/module.xml

```diff
@@ -33,9 +33,10 @@
         <module name="java.logging"/>
         <module name="java.sql"/>
 
-        <module name="javax.sql.api"/>
+        <!--module name="javax.sql.api"/-->
```

Comment:
What's the purpose of commenting this out?

Reply:
This was me testing to see if this was still necessary (it isn't).

Reply:
Yeah, drop!